src/components/Course/Content/Content.js

```diff
@@ -8,4 +9,9 @@ const Content = ({ body, title }) => (
   </div>
 );
 
+Content.PropTypes = {
+  body: PropTypes.string,
```

Comment:
These declarations should have `isRequired` at the end of them.

Reply:
Done.